pkg/sql/tests/rsg_test.go

```diff
@@ -127,13 +127,26 @@ func (db *verifyFormatDB) exec(t *testing.T, ctx context.Context, sql string) er
 func (db *verifyFormatDB) execWithTimeout(
 	t *testing.T, ctx context.Context, sql string, duration time.Duration,
 ) error {
-	if err := verifyFormat(sql); err != nil {
-		db.verifyFormatErr = err
+	defer db.Incr(sql)()
```

Comment:
not a big deal, but looks like this increment used to only happen when there was no error. it's supposed to be tracking actively executing queries